### PR TITLE
Add translation CSV export for locales

### PR DESCRIPTION
**Pull request by kschiffer**
#### Summary
This PR adds a small function to the `tools/mage/translations.js` to export a CSV for translations.

#### Changes
- Add `export-for-translation` flag to export supported locales as CSV

#### Notes for Reviewers
This is useful when creating new translations or updating existing ones:

image

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: tools/mage/translations.js [hunk 1]
```diff
@@ -200,7 +204,7 @@ const readMessages = async function () {
     }, {})
 }
 
-/**
+/** ...................
```
Comment (adriansmares) [inline tools/mage/translations.js]: I guess this is unintentional.

File: tools/mage/translations.js [hunk 2]
```diff
@@ -200,7 +204,7 @@ const readMessages = async function () {
     }, {})
 }
 
-/**
+/** .........
```
Comment (adriansmares) [inline tools/mage/translations.js]: I still see the dots - are they intentional ?

Comment (kschiffer) [reply tools/mage/translations.js]: Hmm, seems to be a bug with my formatter. I'll check again.

Comment (kschiffer) [reply tools/mage/translations.js]: Now gone.

**Review by adriansmares — APPROVED** [review]

**Pull request merged**